deploy/crds/toolchain_v1alpha1_usersignup.yaml

```diff
@@ -11,6 +11,10 @@ spec:
     name: User ID
     priority: 1
     type: string
+  - JSONPath: .spec.username
+    name: Username
+    priority: 1
+    type: string
```

Comment:
this should be generated from `api` repo - see my comment there: https://github.com/codeready-toolchain/api/pull/31#pullrequestreview-282615206

Reply:
When I run `make generate` in the api project, I get the following error:

`The local '../host-operator' repository has pending changes. Please stash them or commit them, first.
`

There are no uncommitted changes, any idea what this means?

Reply:
Nevermind, I worked it out.. simply having a file in the host-operator directory (I usually pipe my test output to a text file) was enough to break this make target, even though the file wasn't under git version control.  I've regenerated the crd and added the changes to the PR.

Reply:
well, the make target checks the git status using `git status -s ` so if the file was detected, then the target failed as it expected a clear environment